config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openstack-k8s-operators/manila-operator
-  newTag: latest
+  newName: quay.io/fpantano/manila-operator
```

Comment:
we'll want to remove this before merging

Reply:
Done